dist/auth/google.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
```

Comment:
Hi. I am not sure your dist/ folder should have been uploaded, can you double-check that and your gitignore file. Thanks